src/components/Pages/Service.js

```diff
@@ -68,26 +69,31 @@ const Service = ({ service, intl }) => {
                     )
                   ) : null}
 
+                  {!!dynamicContent &&
+                    dynamicContent.map(content => (
```

Comment:
What is the significance of the `!!` here? The way I understand it, `!!dynamicContent == dynamicContent`

Reply:
Would we also want to check that dynamicContent has a "length" like we do for contacts?

Reply:
The short answer is that this is just adding back code that already was here before and worked. 

Longer answer; it's a way of checking if the object exists:
https://medium.com/better-programming/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054